docs/reST/ref/math.rst

```diff
@@ -1017,6 +1053,10 @@ Multiple coordinates can be set using slices or swizzling
 
       .. ## Vector3.update ##
 
+   .. attribute:: epsilon
+
+      | :sl:`With lengths within this number, vectors are considered equal. For more information see` Vector2_:
```

Comment:
This does generate a link to Vector.epsilon, but how?

It seems like it would be preferable to have an explicit reference to Vector2.epsilon.